rs/bitcoin/ckbtc/minter/tests/tests.rs

```diff
@@ -422,6 +423,14 @@ fn test_no_new_utxos() {
             suspended_utxos: Some(vec![]),
         })
     );
+    ckbtc
+        .check_minter_metrics()
+        .assert_contains_metric_matching(
+            r#"ckbtc_minter_update_calls_latency_bucket\{new_utxos="no",le="\+Inf"\} 1 \d+"#,
```

Comment:
nit: not that if you only need to check a substring (to ignore the timestamp) at the end, using [`String::contains`](https://doc.rust-lang.org/std/string/struct.String.html#method.contains) or `String::starts_with` might be enough. Also ok to keep the `regex` dependency to have more flexibility here.